2-collectors/scala-stream-collector/config.hocon.sample

```diff
@@ -0,0 +1,165 @@
+      #
+      # If both are set to 'iam', use AWS IAM Roles to provision credentials.
+      #
+      # If both are set to 'env', use environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+      aws {
+        access-key: "iam"
+        secret-key: "iam"
+      }
+
+      # Data will be stored in the following stream.
+      stream {
+        region: "{{collectorSinkKinesisStreamRegion}}"
+        good: "{{collectorKinesisStreamGoodName}}"
+        bad: "{{collectorKinesisStreamBadName}}"
+      }
+
+      # Minimum and maximum backoff periods
+      backoffPolicy: {
+        minBackoff: 100
+        maxBackoff: 200
+      }
+    }
+
+    kafka {
+      brokers: "{{collectorKafkaBrokers}}"
+
+      # Data will be stored in the following topics
+      topic {
+        good: "{{collectorKafkaTopicGoodName}}"
+        bad: "{{collectorKafkaTopicBadName}}"
+      }
+    }
+
+    gcpubsub {
+      # this is the project ID inside of which your collector will be running
+      googleAuthPath: "snowflake-test.json"
+      googleProjectId: "snowflake-test"
+
+      # Data will be stored in the following topics (they must have been created ahead-of-time)
+      topic {
+        good: "snowplow-collector-good"
+        bad: "snowplow-collector-bad"
+      }
+
+      # Minimum and maximum backoff periods and retry multiplier
+      retryPolicy: {
+        minBackoff: 100
+        maxBackoff: 400
+        retryDelayMultiplier: 2.0
+        totalTimeOut: 10000
+        initialRpcTimeout: 1000
+        maxRpcTimeout: 1000
+      }
+    }
+
+    # Incoming events are stored in a buffer before being sent to Kinesis/Kafka/PubSub.
+    # The buffer is emptied whenever:
+    # - the number of stored records reaches record-limit or
+    # - the combined size of the stored records reaches byte-limit or
+    # - the time in milliseconds since the buffer was last emptied reaches time-limit
+    buffer {
+      byte-limit: 100000
+      record-limit: 1 # Not supported by Kafka; will be ignored
+      time-limit: 60000
+    }
+  }
+}
+
+# Akka has a variety of possible configuration options defined at
+# http://doc.akka.io/docs/akka/2.2.3/general/configuration.html.
+akka {
+  loglevel = DEBUG # 'OFF' for no logging, 'DEBUG' for all logging.
+  loggers = ["akka.event.slf4j.Slf4jLogger"]
+}
+
+# spray-can is the server the Stream collector uses and has configurable
+# options defined at
+# https://github.com/spray/spray/blob/master/spray-can/src/main/resources/reference.conf
+spray.can.server {
+  # To obtain the hostname in the collector, the 'remote-address' header
+  # should be set. By default, this is disabled, and enabling it
+  # adds the 'Remote-Address' header to every request automatically.
+  remote-address-header = on
+
+  uri-parsing-mode = relaxed
+  raw-request-uri-header = on
+
+  # Define the maximum request length (the default is 2048)
+  parsing {
+    max-uri-length = 32768
+  }
+}
```

Comment:
please modify the existing config: https://github.com/snowplow/snowplow/blob/master/2-collectors/scala-stream-collector/examples/config.hocon.sample